pbrd/pbr_map.c

```diff
@@ -114,6 +118,26 @@ static bool pbrms_is_installed(const struct pbr_map_sequence *pbrms,
 	return false;
 }
 
+void pbr_set_match_clause_for_pcp(struct pbr_map_sequence *pbrms, uint8_t pcp)
+{
+	if (pbrms) {
+		pbrms->match_pcp = pcp;
+		zlog_info("setting pbrms->match_pcp = %u ", pbrms->match_pcp);
+
+		pbr_map_check(pbrms, true);
+	}
+}
```

Comment:
How does this work for matching PCP==0 ? Unless I missed something in the code, there is no use of a separate flag to indicate that PCP should be matched vs. ignored.